compiler-core/src/parse.rs

```diff
@@ -2819,12 +2842,27 @@ where
         parser: &impl Fn(&mut Self) -> Result<Option<A>, ParseError>,
         sep: Option<&Token>,
     ) -> Result<Vec<A>, ParseError> {
+        let (_, res) = self.sep_series_of(parser, sep)?;
+        Ok(res)
+    }
+
+    // Parse a series by repeating a parser, and a separator. Returns true if
+    // the series ends with the trailing separator.
+    fn sep_series_of<A>(
```

Comment:
What's a sep? No abbreviations please 🙏 

Also `///` for documentation comments.